tutorials/introduction-gee-python-api-guiattard/tutorial.ipynb

```diff
@@ -0,0 +1,1244 @@
+{
```

Comment:
<ul><li>Sp. Pyhton &gt; Python</li><li>Caps: Google Earth <strong>engine</strong> &gt; Google Earth <strong>Engine</strong></li><li>Sp. Thank's &gt; Thanks</li><li>reviewing and helping in writting &gt; reviewing and helping write</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/254/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>Done</p>